docs/utils/cookies.md

```diff
@@ -28,7 +28,7 @@ export const userPrefs = createCookie("user-prefs", {
 
 Then, you can `import` the cookie and use it in your `loader` and/or `action`. The `loader` in this case just checks the value of the user preference so you can use it in your component for deciding whether to render the banner. When the button is clicked, the `<form>` calls the `action` on the server and reloads the page without the banner.
 
-**Note:** We recommend (for now) that you create all the cookies your app needs in `app/cookies.js` and `import` them into your route modules. This allows the Remix compiler to correctly prune these imports out of the browser build where they are not needed. We hope to eventually remove this caveat.
+**Note:** We recommend (for now) that you create all the cookies your app needs in `app/cookies.server.ts` and `import` them into your route modules. This allows the Remix compiler to correctly prune these imports out of the browser build where they are not needed. We hope to eventually remove this caveat.
```

Comment:
```suggestion
**Note:** We recommend (for now) that you create all the cookies your app needs in a `*.server.ts` file and `import` them into your route modules. This allows the Remix compiler to correctly prune these imports out of the browser build where they are not needed. We hope to eventually remove this caveat.
```
This way it doesn't sound like using `cookie.server.ts` is special, the important part is `.server.ts` not the file name itself

Reply:
@sergiodxa oh isn't it? 😅 I did actually thought that it was there 🙈 

Thanks for the suggestion 🙌